target/linux/ath79/generic/base-files/lib/preinit/10_fix_eth_mac.sh

```diff
@@ -14,7 +14,8 @@ preinit_set_mac_address() {
 	enterasys,ws-ap3705i)
 		ip link set dev eth0 address $(mtd_get_mac_ascii u-boot-env0 ethaddr)
 		;;
-	siemens,ws-ap3610)
+	siemens,ws-ap3610|\
+	extreme,ap3805i)
 		ip link set dev eth0 address $(mtd_get_mac_ascii cfg1 ethaddr)
```

Comment:
Is this required? Otherwise, using lan_mac in 02_network is preferred.

Reply:
Setup in preinit is preferred, as this way the config does not contain static device information and is portable between devices.

Assigning the MAC address to a device using netifd is only really necessary when dealing with different MAC addresses on a single master interface.

Reply:
I just followed the example of the Enterasys WS-AP3705i. That one had them both filled in and i assumed preinit was the main approach with 02_network as redundancy?
So should any be removed?

Reply:
writing the MAC address to 02_network is only necessary in case the LAN / WAN interfaces share a single master (e.g. they are separate VLAN on the same MAC).

Applying the MAC address in the preinit phase is preferred on ath79, as it more closely resembles the behavior of the `mtd-mac-address` DTS-way and keeps the configuration portable across devices.

Reply:
Fixed